py/server/deephaven/stream/__init__.py

```diff
@@ -11,6 +11,27 @@
 from deephaven.table import Table
 
 _JBlinkTableTools = jpy.get_type("io.deephaven.engine.table.impl.BlinkTableTools")
+_JAddOnlyToBlinkTableAdapter = jpy.get_type("io.deephaven.engine.table.impl.AddOnlyToBlinkTableAdapter")
+
+def add_only_to_blink(table: Table) -> Table:
+    """ Creates a blink table from the given add-only table.
+
+    Note that the use of this function should be limited to add-only tables that are not fully in-memory, or when
```

Comment:
why not in-memory?

Reply:
My reasoning is that if the upstream table is fully in memory, they aren't going to be much benefit from creating a downstream blink table, @rcaudy?

Reply:
Correct. It's basically a dumb thing to do with no benefit and material cost, unless you're an expert with a good reason.

Reply:
The doc string is not clear about the reason.  It isn't clear if it is dumb or just doesn't work.